curriculum/challenges/english/20-upcoming-python/learn-algorithm-design-by-building-the-shortest-path-algorithm/655773f8b8b5db9fc6d0ae76.md

```diff
@@ -0,0 +1,56 @@
+---
+id: 655773f8b8b5db9fc6d0ae76
+title: Step 10
+challengeType: 20
+dashedName: step-10
+---
+
+# --description--
+
+To keep track of the visited nodes, you need a list made of all the nodes in the graph. Once a node is visited, it will be removed from that list.
```

Comment:
```suggestion
To keep track of the visited nodes, you need a list of all the nodes in the graph. Once a node is visited, it will be removed from that list.
```